polyfill/test/ecmascript.mjs

```diff
@@ -0,0 +1,196 @@
+import Demitasse from '@pipobscure/demitasse';
+const { describe, it, report } = Demitasse;
+
+import Pretty from '@pipobscure/demitasse-pretty';
+const { reporter } = Pretty;
+
+import Assert from 'assert';
+const { ok: assert, deepEqual, throws } = Assert;
+
+import { ES } from '../lib/ecmascript.mjs';
+import { GetSlot, IDENTIFIER } from '../lib/slots.mjs';
+
+describe('ECMAScript', () => {
+    describe('GetTimeZoneDateTimeParts', () => {
+        describe('epoch', () => {
+            test(0n, GetSlot(ES.ToTimeZone('America/Los_Angeles'), IDENTIFIER), {
```

Comment:
It seems unexpected that tests refer to internal state of objects - what's up with that?

Reply:
Because `test/ecmascript.mjs` is testing the internal algorithms in `lib/ecmascript.mjs` rather than publically available things. These are actually unit tests, where the unit is exposed only internally.